requirements.txt

```diff
@@ -1,9 +1,13 @@
-pydocstyle==3.*
+# Docker
+docker==4.0.*
+
+# Style-check
+pydocstyle==4.0.0
```

Comment:
Did you solve those problem with pydocstyle version?

Reply:
This versions together work fine